pydm/widgets/display_format.py

```diff
@@ -12,13 +13,20 @@ class DisplayFormat(object):
 
 
 def parse_value_for_display(value, precision, display_format_type=DisplayFormat.Default, string_encoding="utf_8", widget=None):
+    try:
```

Comment:
Replace this try/except with:
```python
    widget_name = "" if widget is None else widget.objectName()
```

Reply:
type(widget) shows there's a pyqtWrapperType around the QWidget:
>{type 'PyQt5.QtCore.pyqtWrapperType}<class 'PyQt5.QtWidgets.QWidget'>

which doesn't have the objectName method. That's why I catch the AttributeException.